src/labelformat/formats/yolov8.py

```diff
@@ -69,8 +69,13 @@ def _root_dir(self) -> Path:
         return self._config_file.parent
 
     def _images_dir(self) -> Path:
+        """Get images directory from YOLOv8 config file with fallback logic."""
         root_dir = self._root_dir()
-        return root_dir / str(self._config_data[self._split])
+        split_path = str(self._config_data[self._split])
+        # Fix relative paths for Roboflow-style configs.
+        if "path" not in self._config_data and split_path.startswith("../"):
+            split_path = split_path.replace("../", "./", 1)
```

Comment:
This is not the same fix as [used by ultralytics](https://github.com/ultralytics/ultralytics/blob/da9d8730f9f8a99fe6cd980fc927fd1005c15676/ultralytics/data/utils.py#L312). They first check if the `../` path exists and only if it doesn't they replace it with the `./`. I.e. a valid `../` path works.
What ist the reason for making it differently?